doc/CLI.rst

```diff
@@ -44,7 +44,7 @@ Pythran can also generate raw C++ code, using the ``-e`` switch::
 
   $> pythran -e cli_foo.py -o cli_foo.hpp
   $> printf '#include \"cli_foo.hpp\"\nusing namespace __pythran_cli_foo ; int main() { foo()(); return 0 ; }' > cli_foo.cpp
-  $> `pythran-config --compiler` cli_foo.cpp `pythran-config --cflags --libs` -o cli_foo
+  $> `pythran-config --compiler` -std=c++11 cli_foo.cpp `pythran-config --cflags --libs` -o cli_foo
```

Comment:
Looking at the pythran.cfg file, -std=c++11 is already in the cflags sections and pythran-config is loading this information. Here is a duplication.
Users can change it updating the .pythranrc file.